src/OnlineSales/Services/EmailFromTemplateService.cs

```diff
@@ -0,0 +1,60 @@
+﻿// <copyright file="EmailFromTemplateService.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using Microsoft.EntityFrameworkCore;
+using OnlineSales.Data;
+using OnlineSales.Entities;
+using OnlineSales.Interfaces;
+
+namespace OnlineSales.Services
+{
+    public class EmailFromTemplateService : IEmailFromTemplateService
+    {
+        private readonly IEmailWithLogService emailWithLogService;
+        private readonly ApiDbContext apiDbContext;
+
+        public EmailFromTemplateService(IEmailWithLogService emailWithLogService, ApiDbContext apiDbContext)
+        {
+            this.emailWithLogService = emailWithLogService;
+            this.apiDbContext = apiDbContext;
+        }
+
+        public async Task SendAsync(string templateName, string recipient, params object[] templateArguments)
```

Comment:
@gsplakmal please use Dictionary<string, string> type for templateArguments parameter.

Reply:
Updated with a dictionary parameter